ios/GitPoint.xcodeproj/project.pbxproj

```diff
@@ -5,7 +5,6 @@
 	};
 	objectVersion = 46;
 	objects = {
-
```

Comment:
@SpaceLenore If you can revert this needless change when resolving conflicts, it will be better. Then we can merge it!

Reply:
@chinesedfan updated with a commit to undo it. Don't know why it happened, it was automated somewhere. Fixed now anyways